js/main.js

```diff
@@ -19,23 +17,88 @@ $(function() {
   //get questions from an API
   function getQuestions(difficulty) {
     switch (difficulty) {
-
       case "easy":
-        $.get("https://opentdb.com/api.php?amount=50&category=9&difficulty=easy&type=multiple", function(data){
-          return data;
+          $.get("https://opentdb.com/api.php?amount=50&category=9&difficulty=easy&type=multiple", function(data){
+          questions = data;
+          return questions;
         })
         break;
       case "medium":
-        $.get("https://opentdb.com/api.php?amount=50&category=9&difficulty=medium&type=multiple", function(data){
-          return data;
-
+          $.get("https://opentdb.com/api.php?amount=50&category=9&difficulty=medium&type=multiple", function(data){
+          questions = data;
+          return questions;
         })
         break;
       case "hard":
         $.get("https://opentdb.com/api.php?amount=50&category=9&difficulty=hard&type=multiple", function(data){
-          return data;
+          questions = data;
+          debugger;
```

Comment:
need to remove 
debugger;
line of code before commit

Reply:
Hi Anthony, I removed the debugger
Thank you